physics/samfdeepcnv.f

```diff
@@ -222,7 +222,7 @@ subroutine samfdeepcnv_run (im,ix,km,itc,ntc,cliq,cp,cvap,        &
       parameter(clamd=0.03,tkemx=0.65,tkemn=0.05)
       parameter(dtke=tkemx-tkemn)
       parameter(dbeta=0.1)
-      parameter(cthk=200.,dthk=25.)
```

Comment:
The different cthk may change results.

Reply:
Good catch @mzhangw ! But we should take this change from master, right?